sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -40,11 +40,11 @@ public void setUser(User user) {
     this.user = user;
   }
 
-  public List<String> getFingerprint() {
+  public List<String> getFingerprints() {
```

Comment:
Sorry it's confusing but the API is `getFingerprint`:

```
scope.set_fingerprint(fingerprint[]): Sets the fingerprint to group specific events together
```

https://docs.sentry.io/development/sdk-dev/unified-api/#scope

A way to look at it is that "A fingerprint is a collection of strings`.

```suggestion
  public List<String> getFingerprint() {
```

Reply:
fixed, the whole docs but the method is called `fingerprints` (plural).
https://docs.sentry.io/data-management/event-grouping/